Library/Formula/wslay.rb

```diff
@@ -0,0 +1,64 @@
+class Wslay < Formula
+  homepage "http://wslay.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/wslay/wslay-1.0.0/wslay-1.0.0.tar.xz"
+  sha1 "199308322e67094ee803063eca0077dfc042bc77"
+
+  option "without-docs", "Don't generate or install documentation"
+
+  head do
+    url "https://github.com/tatsuhiro-t/wslay.git"
+
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on :python => :build if MacOS.version <= :snow_leopard && build.with?("docs")
+  depends_on "cunit" => :build
```

Comment:
Already fixed locally, I just hadn't pushed the follow up PR yet. Got distracted working on something much bigger and something I'm inevitably likely to end up hating myself for starting :grinning:. Will push the local PR now.